editor.planx.uk/src/@planx/components/MapAndLabel/Public/index.tsx

```diff
@@ -61,31 +56,46 @@ const VerticalFeatureTabs: React.FC<{ features: Feature[] }> = ({
         maxHeight: "fit-content",
       }}
     >
-      <TabContext value={activeTab}>
+      <TabContext value={activeIndex.toString()}>
         <Tabs
           orientation="vertical"
           variant="scrollable"
-          value={activeTab}
-          onChange={handleChange}
+          value={activeIndex.toString()}
+          onChange={(_e, newValue) => {
+            editFeature(parseInt(newValue, 10));
+          }}
+          // TODO!
           aria-label="Vertical tabs example"
           sx={{ borderRight: 1, borderColor: "divider" }}
         >
           {features.map((feature, i) => (
             <Tab
               key={`tab-${i}`}
-              value={feature.properties?.label}
+              value={i.toString()}
               label={`${schema.type} ${feature.properties?.label}`}
               {...a11yProps(i)}
+              {...(isFeatureInvalid(i) && {
+                sx: (theme) => ({
+                  backgroundColor: theme.palette.action.focus,
+                }),
+              })}
```

Comment:
This is how we get yellow tabs on error - styling very much TBC here.

Reply:
I think a simple style option here is to make the default `borderRight` on the tab our `error` color and double the thickness when invalid? But again, super easy to fix forward / adjust later - thanks for highlighting placeholder here !

Reply:
That would work well! I've added a styling item to https://trello.com/c/JafdzU94/2855-epic-allow-applicants-to-work-to-trees-to-see-and-add-more-information-about-protected-trees-on-a-map to make sure this gets picked up here.

Going with the fix-forward approach due to slightly limited time this week and other PRs which are based on this branch 👍